src/w3c/headers.js

```diff
@@ -537,6 +537,9 @@ export function run(conf) {
       "If one of '`wg`', '`wgURI`', or '`wgPatentURI`' is an array, they all have to be."
     );
   }
+  if (conf.isCGBG && !conf.wg) {
+    pub("error", "`respecConfig.wg` is required.");
```

Comment:
Nit: link to wiki for `wg`

Reply:
might want to give a little more help here too.  "`wg` configuration option is required for CG docs (e.g., "Web Incubator Community Group")."